docs/concepts.rst

```diff
@@ -955,6 +955,19 @@ a re-imported version of the flow.  (This is a fairly magical procedure -- in
 complicated cases, it may not be able to figure out how to do this.  In these
 cases it will try to throw an exception rather than fail silently.)
 
+You can also use the :meth:`Flow.reloading_in_place <bionic.Flow.reloading_in_place>`
```

Comment:
We'll want to update the documentation to reflect the new method name, and the fact that `reload` is now the primary way of doing a reload.